doc/auto-discovery-docker.md

```diff
@@ -102,6 +102,7 @@ services:
     network_mode: host
     environment:
       SERVICE_HOST: 192.168.56.101
+      SERVICE_SERVERBINDADDR: 0.0.0.0
```

Comment:
This open the service to any external traffic and is not recommended for secure-mode.

Reply:
The two things above do that mostly already. `network_mode: host` and `SERVICE_HOST: 192.168.56.101`. If this was not network mode host, the `0.0.0.0` was not needed for the use-case in which it was added.

Reply:
I will try to improve auto-discovery at https://github.com/edgexfoundry-holding/device-onvif-camera/issues/39.

@ajcasagrande Is SERVICE_SERVERBINDADDR used for set-credentials.sh?

Reply:
@weichou1229 It needs it only when the service host is not localhost.

Reply:
In my other comment I explain how if you change the service host then you can't access it via localhost anymore and need to use the host ip. The 0.0.0.0 was to let my script still call it via localhost.

Reply:
OK, so we can remove that after I complete the improvement.

Reply:
Intel team is working on the discovery improvement https://github.com/edgexfoundry-holding/device-onvif-camera/issues/39#issuecomment-1103177166